Composer/packages/extensions/adaptive-form/src/utils/resolveRef.ts

```diff
@@ -8,6 +8,10 @@ export function resolveRef(
   definitions: { [key: string]: JSONSchema7Definition } = {}
 ): JSONSchema7 {
   if (typeof schema?.$ref === 'string') {
+    if (!schema?.$ref?.startsWith('#/definitions/')) {
```

Comment:
why was this needed?

Reply:
Some of the refs in the Dialog Schema refer to 'http://json-schema.org/draft-07/schema#', so it throws the function throws an error since 'http://json-schema.org/draft-07/schema#' is not included in the `definitions`.